lib/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] I also noticed in your `Book class` that per the project requirements, you needed to:

> Create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. The following should be implemented:
> Create the has-many side of Book and Rental (a book has many rentals)

- Currently, this isn't being implemented in the `Book class`. I will suggest you include a method for adding rentals in the class. The book itself should be sent as a parameter to create the new rental thereby achieving the `has-many` association. :wink: 
- Here is a [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship%20def%20add_rental(person,%20date)) to an example of this implementation in a previous lesson :+1: 
- Example implementation

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```